src/generators/TypescriptInterfaceGenerator.test.js

```diff
@@ -50,7 +50,9 @@ test("Generate a typescript interface", () => {
 
   expect(fs.existsSync(tmpobj.name + "/interfaces/foo.ts")).toBe(true);
 
-  const res = `export interface Foo {
+  const res = `import { FooBar } from "./foobar";
+
+export interface Foo {
```

Comment:
indent maybe ?

Reply:
You mean indent before the `export` line? I think this is correct as it is.

As a reference, this is the output I get on an API of mine

```typescript
import { Company } from "./company";
  
export interface Client {
  '@id'?: string;
  id: string;
  name?: string;
  legalName?: string;
  legalAddressCity?: string;
  legalAddressStreet?: string;
  legalAddressStreetNumber?: string;
  legalAddressZipcode?: string;
  phone?: string;
  email?: string;
  vatNumber?: string;
  taxNumber?: string;
  readonly updatedAt?: Date;
  readonly createdAt?: Date;
  readonly agency?: Company;
  readonly locationsCount?: number;
}

```

Reply:
i mean for readability, it's in the function test body so it shouldn't be on the same level (line 8)

Reply:
also line 108 to 113 need a small indent :)

Reply:
If I apply the indentation I get this
```typescript
test("Generate a typescript interface", () => {
  [...]
  const res = `import { FooBar } from "./foobar";

  export interface Foo {
    '@id'?: string;
    id: string;
    foo: any;
    foobar?: FooBar;
    readonly bar: string;
  }
  `;
  [...]
}
```
however the test fails though, because it adds an indent to every line of the expected result.

Maybe we should define some output files with the expected output and we read them during the test? That way the read operation could be indented without issues

Reply:
oh didn't thought of this issue, so i think it's ok like that ! (without output files ^^)

Reply:
Ok!